.github/CODEOWNERS

```diff
@@ -79,10 +79,10 @@
 /sdk/resourcemanager/         @ArcturusZhang @lirenhe @tadelesh @raych1
 
 # AzureSDKOwners: @jhendrixMSFT
-# ServiceOwner: @siminsavani-msft
+# ServiceOwner: @tanyasethi
```

Comment:
This should be the GirHub alias. Isn't that @tanyasethi-msft ?

Reply:
Apologies, here's the corrected PR - https://github.com/Azure/azure-sdk-for-go/pull/23216